docs/release-strategy.md

```diff
@@ -65,7 +65,7 @@ The following are the steps for how Y-stream and Z-stream releases gets cut.
 1. Validate the release branch with an [E2E test](ci.md).
 1. Create a new release on GitHub targeting the release branch and using the previous Z-Stream tag as the previous release (e.g. `0.15.0` precedes `0.15.1`).
 1. Move the `stable` tag to the new release (note this tag is set to be deprecated on September 1st, 2024 - users should use PyPi to install the latest "stable" release)
-1. Announce release via the following:
+1. Announce the release if necessary. Not all Z-Streams necessitate an email or slack message if the number of commits is low or if the content of the commits is not consequential to most users. If warranted announce on:
```

Comment:
```suggestion
1. Announce the release if the Maintainer team deems it is necessary. If desired announce on:
```

Reply:
I think a mix of both of these wordings makes sense.

Adding what is "necessary" and adding that the maintainers deem whether it is necessary or not makes sense to me

Reply:
Why back ourselves into a corner unnecessarily?

Also fine with the wording @khaledsulayman suggested below